cdap-docs/application-templates/source/etl/templates/sinks/batch/table.rst

```diff
@@ -6,13 +6,49 @@
 Sinks: Batch: Table 
 ===============================
 
-.. rubric:: Description: Batch Sink for CDAP Tables
+.. rubric:: Description
 
-**Name:** Name of the table dataset. If it does not already exist, one will be  created.
+Batch Sink for CDAP Tables. Writes records to a Table with one record field mapping
```

Comment:
in any case, I think I will just remove the first sentence because it does not convey any useful information